fmriprep/cli/parser.py

```diff
@@ -40,6 +40,27 @@ def _build_parser(**kwargs):
 
     from .version import check_latest, is_flagged
 
+    deprecations = {
+        # parser attribute name: (replacement flag, version slated to be removed in)
+        'use_aroma': (None, '24.0.0'),
+        'aroma_melodic_dim': (None, '24.0.0'),
+        'aroma_err_on_warn': (None, '24.0.0'),
```

Comment:
24.0 is next. We can just delete these ones...

Reply:
I figured we could remove these (along with the arguments themselves) in a single, separate PR